src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -1621,18 +1621,21 @@ def numpy_to_pil(images):
         """
         return numpy_to_pil(images)
 
-    def progress_bar(self, iterable=None, total=None):
+    def progress_bar(self, iterable=None, total=None, desc=None, leave=True):
```

Comment:
There are two nested progress bars in the `__call__` method. Do you suggest adding a call to `set_progress_bar_config` before every usage of `self.progress_bar` withing the `__call__` method? Seems rigid, not to mention that this will interfere with tqdm flags set by the user. My proposed code gives priority to whatever flags the user chose to set.

Reply:
can we do that? run `set_progress_bar_config` before every use?

Reply:
another method is to create a `progress_bar` method just for marigold pipeline to override the one on the base class

Reply:
> another method is to create a progress_bar method just for marigold pipeline to override the one on the base class

Yep let's do that!

Reply:
Done